src/lib.rs

```diff
@@ -1215,11 +1215,15 @@ impl Certificate {
 		Ok(pem::encode(&p))
 	}
 	/// Serializes the private key in PKCS#8 format
+	///
+	/// Panics if called with on a remote key pair.
```

Comment:
```suggestion
	/// Panics if called on a remote key pair.
```

Reply:
Sorry about that, I wasn't actually done going through it again!